docs/src/main/sphinx/installation/cli.rst

```diff
@@ -116,6 +130,11 @@ JWT authentication
 To access a Trino cluster configured to use :doc:`/security/jwt`, use the
 ``--access-token=<token>`` option to pass a JWT to the server.
 
+Kerberos authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using Kerberos, see :doc:`/security/cli`
```

Comment:
If your cluster is secured with Kerberos authenticatiomn , see LINK  for more details.